src/codec_svt.c

```diff
@@ -73,7 +73,9 @@ static avifResult svtCodecEncodeImage(avifCodec * codec,
         }
         svt_config->encoder_color_format = color_format;
         svt_config->encoder_bit_depth = (uint8_t)image->depth;
+#if SVT_VERSION_PATCHLEVEL < 8
```

Comment:
We also need to test SVT_VERSION_MAJOR and SVT_VERSION_MINOR. In pseudo code, the correct check is

`#if SVT_VERSION < 0.8.8`

I took a quick look. We should be able to use the SVT_CHECK_VERSION(major,minor,patch) macro. So I think this test should read:

`#if !SVT_CHECK_VERSION(0, 8, 8)`

Reply:
I just realized that I found the SVT_CHECK_VERSION macro in SVT-VP9, not SVT-AV1:

https://github.com/OpenVisualCloud/SVT-VP9/blob/master/Source/API/EbApiVersion.h.in

SVT-AV1 does not seem to have an equivalent macro. So we will need to define a similar macro ourselves or use some other approach.

Reply:
@wantehchang https://gitlab.com/AOMediaCodec/SVT-AV1/-/merge_requests/1783, I don't know if you have a gitlab account I can ping, so I will mention it here

Reply:
@1480c1 Thanks a lot! I assume you prefer SVT_AV1_CHECK_VERSION over a packed-version SVT_AV1_VERSION macro.

Reply:
> @1480c1 Thanks a lot! I assume you prefer SVT_AV1_CHECK_VERSION over a packed-version SVT_AV1_VERSION macro.

Indifferent right now, since I'm not entirely sure on what the limits of the packed version check would be, but I assume if we manage to go to `0.9999999.1` in svt-av1, then it might break